README.md

```diff
@@ -1,2 +1,13 @@
 # bench-uperf
-Helper scripts to aid the execution and post-processing of uperf
+Helper scripts to aid the execution and post-processing of uperf network performance tool.
+
+## Scripts
+
+Name | Description
+-----|------------
+uperf-base | Checks if the bench-base library can be sourced from the ${TOOLBOX_HOME}/bash/library/ directory. If the library cannot be sourced, the script prints an error message and exits with an exit status of 1.
+uperf-client | Runs a network benchmark using uperf, a network performance tool. The script has several command-line options that can be used to customize the benchmark run, including the number of threads, protocol, packet sizes, and duration.
+uperf-get-runtime | Outputs the duration of a benchmark run.
+uperf-post-process | Perl script that processes results from the uPerf benchmark tool and outputs a JSON file containing performance metrics.
```

Comment:
Let's go ahead and reference that these JSON documents adhere to the CDM (CommonDataModel) and include a link to that repository.

Reply:
Except that it is our CDM repo, not microsoft's :)

https://github.com/perftool-incubator/CommonDataModel

Reply:
Fixed!